src/index.js

```diff
@@ -15,6 +16,45 @@ const KeyCommand = NativeModules.KeyCommand ? NativeModules.KeyCommand : new Pro
     },
 );
 
+function validateKeyCommand(keyCommand) {
+    /**
+     * command object may have following schema:
+     * - {input: 'f', modifierFlags: 123}
+     * - {input: 'f'}
+     * - {input: '123'}
+     */
+    if (!keyCommand.input) {
+        // eslint-disable-next-line
+        console.error('input property for keyCommand object must be an provided. skipping.');
+        return;
+    }
+
+    if (keyCommand.input && typeof keyCommand.input !== 'string') {
+        return {...keyCommand, input: `${keyCommand.input}`};
+    }
+
+    return keyCommand;
+}
+
+/**
+ * Validates key command combination list.
+ *
+ * @param {Object[]} keyCommands - List of key command objects.
+ * @param {string} keyCommands[].input - any character key from the keyboard.
+ * @param {number} keyCommands[].modifierFlags - predefined command from getConstants enum.
+ * @returns {Promise}
+ */
+function validateKeyCommands(keyCommands) {
+    const validatedKeyCommands = _.map(keyCommands, validateKeyCommand);
+    const filteredKeyCommands = _.filter(validatedKeyCommands, item => item);
```

Comment:
```suggestion
```

Instead of this,  add the check for not empty in validateKeyCommand.

Reply:
removed